pyanaconda/ui/gui/spokes/lib/refresh.glade

```diff
@@ -265,9 +266,9 @@ installation options while this scan completes.</property>
                       <object class="GtkLabel" id="label12">
                         <property name="visible">True</property>
                         <property name="can_focus">False</property>
-                        <property name="xalign">0</property>
                         <property name="label" translatable="yes">Pressing 'OK' below will take you to the disk selection screen where you will need to re-select your disks.</property>
                         <property name="wrap">True</property>
+                        <property name="xalign">0</property>
```

Comment:
Looking at this change and many of the above changes -- would you mind separating the two steps of "*save everything with new glade*" and "*do the changes*" into two separate commits?